lib/base/scriptutils.cpp

```diff
@@ -520,7 +520,7 @@ String ScriptUtils::MsiGetComponentPathShim(const String& component)
 
 Array::Ptr ScriptUtils::TrackParents(const Object::Ptr& child)
 {
-	return Array::FromVector(DependencyGraph::GetChildren(child));
+	return Array::FromVector(DependencyGraph::GetParents(child));
```

Comment:
Both Julian and I voted for dropping this entirely. (https://github.com/Icinga/icinga2/pull/10263#discussion_r1882062747)

What you want to introduce here is an even nastier breaking change than dropping the DSL function with v2.15! (Happy debugging to end users. /s)